### ISSUE TITLE
Setting FlashTheme ignores fontStyle, fontSize, fontWeight of contentStyle

### ISSUE THREAD

**Issue by tmaihoff** [sososdk/flash#49]
When I set a `FlashTheme` I expect it to be applied to the FlashBars down the widget tree.

But when I set the `contentStyle`, only the `color` is respected. Other attributes like `fontStyle`, `fontSize` and `fontWeight` are ignored.

See the following example: 

```dart
class FlashTest extends StatelessWidget {
  const FlashTest({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return FlashTheme(
      flashBarTheme: const FlashBarThemeData(
        contentStyle: TextStyle(
          color: Colors.green,
          fontStyle: FontStyle.italic,
          fontSize: 8,
          fontWeight: FontWeight.bold,
        ),
      ),
      child: Builder(
        builder: (context) {
          return TextButton(
            child: const Text('show flashbar'),
            onPressed: () => context.showFlashBar(
              content: const Text('content'),
            ),
          );
        },
      ),
    );
  }
}
```
This produces the following FlashBar. As you can see, only the green color is set:
![WhatsApp Image 2022-07-18 at 17 29 52](https://user-images.githubusercontent.com/44509310/179547130-2858152d-e54d-4abc-a458-64a62a7af5f1.jpeg)

However, when I set the same `contentStyle` directly in the `showFlashBar`method, it works as expected:

```dart
            onPressed: () => context.showFlashBar(
              content: const Text('content'),
              contentStyle: const TextStyle(
                color: Colors.green,
                fontStyle: FontStyle.italic,
                fontSize: 8,
                fontWeight: FontWeight.bold,
              ),
            ),
```
![WhatsApp Image 2022-07-18 at 17 29 52 (1)](https://user-images.githubusercontent.com/44509310/179547145-ee4fd0cb-a7e7-4d7f-94d0-043728460b87.jpeg)

Any idea?

**Comment by sososdk** [comment]
Fixed in `2.0.3+3`.

**Comment by tmaihoff** [comment]
thanks for the quick fix
